tests/InputFiles/ExportCoordinates/Input1D.yaml

```diff
@@ -0,0 +1,18 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+# Executable: ExportCoordinates1D
+# Check: execute
+# ExpectedOutput:
+#   Coordinates0.h5
+
+DomainCreator:
+  Interval:
```

Comment:
It might be convenient to also store the DomainCreator used in generating the coordinates, along with its parameters, so that the coordinates may be reproduced.

Reply:
Right, this will be possible once the input file is stored in the H5, can't find the corresponding issue right now.